sample/shared/src/commonTest/kotlin/co/nimblehq/kmm/template/domain/usecases/UseCaseTest.kt

```diff
@@ -13,33 +13,33 @@ import kotlin.test.BeforeTest
 import kotlin.test.Test
 
 @ExperimentalCoroutinesApi
-class GetUsersUseCaseTest {
+class UseCaseTest {
 
     @Mock
     private val mockRepository: Repository = mock(Repository::class)
 
-    private lateinit var useCase: GetUsersUseCase
+    private lateinit var useCase: UseCase
 
     @BeforeTest
     fun setUp() {
-        useCase = GetUsersUseCaseImpl(mockRepository)
+        useCase = UseCaseImpl(mockRepository)
     }
 
     @Test
-    fun `When calling getUsers successfully - it returns users`() = runTest {
+    fun `When calling successfully - it returns users`() = runTest {
```

Comment:
This is still calling `getUsers`. Wouldn't it be better without changes?

Reply:
@kaungkhantsoe A use case is a single task, so it's implicit to call the implemented function already. We can get rid of the func name in the test case name to reduce manual naming issues 🙏